governance/policies/archiving.md

```diff
@@ -98,6 +100,16 @@ When it has been decided that a set of package versions (aka "versions") should
         - `["(none)"]` the maintainer will not maintain any version
         - `["1.3"]` the maintainer will maintain the latest  version of "1.3.Z"
         - `["2.(latest)"]` the maintainer will maintain the latest minor version specifically of version "2" of the package
+- `x-maintained`:
+    - Allowed values: `true` and `false`
+    - Meaning:
+        - Expresses that this opam package version is maintained (if `true`) or not (if `false`).
+        - Overwrites the `x-maintenance-intent` field
+        - Useful to abandon pre-releases
+        - Also for packages that are used in private opam-repositories or developments and that a maintainer would like to keep in the repository independently of the global package intent
```

Comment:
I am unclear here too. I would not think the opam repository policies would have any bearing on private repositories.

Reply:
I amended the text in df67251905, is this understandable for you now?

Reply:
Yes this helps me a lot. Thanks!